### DISCUSSION TITLE
Send a notification when a new security advisory is reported

### DISCUSSION THREAD

**Discussion by renchap** [community/community#102995] [Code Security]
### Select Topic Area

Product Feedback

### Body

At the moment, if a user reports a security advisory on your repository, there is no notification at all.

You need to either create a Webhook to manually handle this, or remember to check your "Security" tab regularly.

There should be an option to send a notification (+ email if opted-in) to at least the repository owners and, if https://github.com/orgs/community/discussions/63041 is implemented someday, any team defined as default collaborators.

**Comment by 0xE69** [comment]
This is a brilliant suggestion! We defiantly need automatic notifications for security advisories on repositories. Currently, repository owners must rely on manual checks or Webhooks, which is not ideal. The proposal to send automatic notifications (and emails for those who opt-in) to repository owners and defined default collaborator teams is a game-changer. It directly supports better operational security by ensuring timely awareness and action on security advisories. Huge thanks to @renchap; We need more tools to sharpen our security posture.

**Comment by DNin01** [comment]
If you want, you can watch a repository by going to its Code tab and clicking "Watch" and select custom notification options. From there, check the box for "Security alerts" and if I am right, you should receive a notification every time a security advisory is published. Does that solve your request?

**Reply by DNin01** [reply]
Now that I think of it, this checkbox might be for code scanning alerts and the like, not advisories, but I'm not sure.
